Can't build. Trying to find nodesass.js within my source folder?

Tried installing with  iojs 3 / npm 2, and node 4 / npm 3. If I try to import a .scss file within my donejs project, this happens when trying to build.

````
Error loading "test@0.0.0#styles.scss!steal-sass@1.0.1#main" from "test@0.0.0#index.stache!done-autorender@0.6.0-pre.0#autorender" at file:/home/nmartel/repos/dev/test/src/index.stache
Error loading "nodesass" at file:/home/nmartel/repos/dev/test/nodesass.js
Error loading "nodesass" from "steal-sass@1.0.1#main" at file:/home/nmartel/repos/dev/test/node_modules/steal-sass/main.js
ENOENT: no such file or directory, open '/home/nmartel/repos/dev/test/nodesass.js'
    at Error (native
````

Is there something to configure with steal-tools?

can't install steal-tools locally (in my project folder) on the vagrant box.

get an error:

```
npm ERR! Linux 3.19.0-25-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "steal-tools"
npm ERR! node v0.12.9
npm ERR! npm  v2.14.9
npm ERR! path ../rimraf/bin.js
npm ERR! code EPROTO
npm ERR! errno -71

npm ERR! EPROTO, symlink '../rimraf/bin.js'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Linux 3.19.0-25-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "steal-tools"
npm ERR! node v0.12.9
npm ERR! npm  v2.14.9
npm ERR! path npm-debug.log.2c37269009e803742bc21afcd75862fe
npm ERR! code ETXTBSY
npm ERR! errno -26

npm ERR! ETXTBSY, rename 'npm-debug.log.2c37269009e803742bc21afcd75862fe'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/vagrant/wecanjs/npm-debug.log
```

--- a/nodesass.js
+++ b/nodesass.js
@@ -1,4 +1,14 @@
 var loader = require("@loader");
-// if we build your app with steal-tools, we are currently not in the steal-sass directory
+var sassSync = "sass.js/dist/sass.sync.js";
+
+// if we are on npm2 and we build your app with steal-tools, we are currently not in the
+// steal-sass directory.
 // we have to set the fully qualified path to the module
-module.exports = loader._nodeRequire(__dirname + "/node_modules/sass.js/dist/sass.sync.js");
+var sassLocation = __dirname + "/node_modules/" + sassSync;
+
+try {
+	// try to resolve the sass module on npm3
+	sassLocation = loader._nodeRequire.resolve(sassSync);
+} catch (e) {}
+
+module.exports = loader._nodeRequire(sassLocation);

--- a/nodesass.js
+++ b/nodesass.js
@@ -1,2 +1,4 @@
 var loader = require("@loader");
-module.exports = loader._nodeRequire("sass.js");
+// if we build your app with steal-tools, we are currently not in the steal-sass directory
+// we have to set the fully qualified path to the module
+module.exports = loader._nodeRequire(__dirname + "/node_modules/sass.js/dist/sass.sync.js");

--- a/package.json
+++ b/package.json
@@ -30,22 +30,22 @@
     "envs": {
       "build": {
         "map": {
-          "$sass.js": "nodesass"
+          "$sass.js": "steal-sass/nodesass"
         }
       },
       "build-development": {
         "map": {
-          "$sass.js": "nodesass"
+          "$sass.js": "steal-sass/nodesass"
         }
       },
       "server-development": {
         "map": {
-          "$sass.js": "nodesass"
+          "$sass.js": "steal-sass/nodesass"
         }
       },
       "server-production": {
         "map": {
-          "$sass.js": "nodesass"
+          "$sass.js": "steal-sass/nodesass"
         }
       }
     },

